### PR TITLE
add velero analyzer docs

### PR DESCRIPTION
**Pull request by arcolife**
Add docs for https://github.com/replicatedhq/troubleshoot/pull/1366

### CODE REVIEW

File: docs/source/analyze/velero.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: Velero
+description: Check backup, restore settings and status.
+---
+
+The `Velero` analyzer is available to check statuses of Custom Resources installed by velero, such as: backup storage locations, backup repositories, backups and restores.
+
+## Parameters
+
+**namespace:** (Optional) Namespace where velero is installed. Will default to `velero` if not set.
```
Comment (banjoh) [inline docs/source/analyze/velero.md:10]: If we chose to remove this in https://github.com/replicatedhq/troubleshoot/pull/1366#pullrequestreview-1710845118, reflect the change here

File: docs/source/analyze/velero.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+**collectorName:** (N/A) Velero currently does not require a special collector as all the Custom Resources are already collected in support bundle.
```
Comment (banjoh) [inline docs/source/analyze/velero.md:12]:
```suggestion
**collectorName:** (N/A) Velero currently does not require a special collector as all the Custom Resources are already collected in support bundle by the `Cluster Resources` collector.
```

File: docs/source/analyze/velero.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+```yaml
+apiVersion: troubleshoot.sh/v1beta2
+kind: SupportBundle
+metadata:
+  name: velero
+spec:
+  collectors:
+    - logs:
+        namespace: velero
+        name: velero/logs
```
Comment (banjoh) [inline docs/source/analyze/velero.md:25]:
```suggestion
spec:
  collectors:
    - clusterResources: {} # implicitly added if not defined in a spec
    - logs:
        namespace: velero
        name: velero/logs
```

**Review by banjoh — COMMENTED** [review]
LGTM aside from the pending decision on what to do with the `namespace`

**Review by banjoh — DISMISSED** [review]
LGTM

**Comment by arcolife** [comment]
@banjoh could you re-approve? removed the doc on `strict` option. Leaving that for the umbrella doc https://troubleshoot.sh/docs/analyze/#strict

**Review by banjoh — APPROVED** [review]

**Pull request merged**